### DISCUSSION TITLE
[Feature Request] Compile included imagemagick with AVIF support

### DISCUSSION THREAD

**Discussion by MaxKorlaar** [beyondcode/herd-community#935] [General]
It seems that the included imagick extension has been compiled with imagemagick that doesn't have support for AVIF enabled. I'd like to use this format but I'm not able to do so at the moment. As imagick comes with the prebuilt static binary I'm not able to override it and swap it out for a custom build of the imagick.so extension that does support it. I'm not sure whether to file this as an issue or here.

<img width="1008" alt="image" src="https://github.com/user-attachments/assets/81884097-f866-4ef5-9622-6e48e9b572a7">

**Comment by RyanRoberts** [comment]
Yeah this is frustrating, it would be good to know if it's either coming or something we could add.

**Comment by robertosanval** [comment]
I have the same problem and it is desperate not to be able to work without the AVIF format that is supported in Imagick.
